.circleci/config.yml

```diff
@@ -414,10 +425,12 @@ jobs:
       # so every node runs exactly one stability test.
       runners-number:
         type: integer 
-        default: 9
+        default: 8
```

Comment:
Please revert it to 9. I introduced a bug in makefile. here is the fix https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/554